docs/src/topics/flavors/cluster-autoscaler.md

```diff
@@ -0,0 +1,33 @@
+# Cluster Autoscaler
+
+## Specification
+
+| Control Plane | CNI    | Default OS   | Installs ClusterClass | Autoscaling |
+|---------------|--------|--------------|-----------------------|-------------|
+| Kubeadm       | Cilium | Ubuntu 22.04 | No                    | Yes         |
+
+## Prerequisites
+
+[Quickstart](../getting-started.md) completed
+
+## Usage
+
+1. Set up autoscaling environment variables
+    ```sh
+    export CLUSTER_AUTOSCALER_VERSION=v1.29.0
+    export WORKER_MACHINE_MIN="\"1\""
```

Comment:
could we move this string escaping into the template instead of the variable?

Reply:
I don't think escaping should be needed, it's already quoted in the template

Reply:
I can give this another shot, but `envsubst` didn't play well with quoting numbers as strings in the templating. It was either stripping them too aggressively or not enough.

Reply:
The issue here seems to be how `clusterctl generate` generates templates in conjunction with `envsubst`. For what I can see, it:

1. Renders and then validates the Kubernetes YAML from the templates
2. Passes the generated YAML through `envsubst`

`envsubst` substitutes both `${var}` and `"${var}"` so you need to explicitly specify the shell variable as a "string".